src/config/index.ts

```diff
@@ -136,8 +136,7 @@ const configMap = {
     appConfig: zimbabweConfig,
     rawLayers: zimbabweRawLayers,
     rawTables: zimbabweRawTables,
-    // TODO - Add selected defaultBoundary to S3
-    defaultBoundariesFile: '',
```

Comment:
Let's use this opportunity to simplify the config for zimbabwe and sri lanka

Reply:
> Let's use this opportunity to simplify the config for zimbabwe and sri lanka

@ericboucher - done